### PR TITLE
make summary site example

### PR DESCRIPTION
**Pull request by D-Petrus**

### CODE REVIEW

File: training-service/src/app/content/summary/summary.component.html [hunk 1]
```diff
@@ -1 +1,18 @@
-<p>TWOJE ZAMÓWIONE KURSY :</p>
+<div class="container">
+    <h1>Podsumowanie</h1>
+    <p>Dziękujęmy za skorzystanie z naszych usług. Twoje zamówione kursy:</p>
+    <div class="form-check">
+    <input class="form-check-input" type="checkbox" value="" id="flexCheckChecked"checked>
```
Comment (holistic-fox) [inline training-service/src/app/content/summary/summary.component.html]: value and checked are not needed attrs here

File: training-service/src/app/content/summary/summary.component.html [hunk 2]
```diff
@@ -1 +1,18 @@
...
+    <label class="form-check-label" for="flexCheckChecked">
+     Kurs 1
+    </label>
+    </div>
+    <div class="form-check">
+    <input class="form-check-input" type="checkbox" value="" id="flexCheckChecked" checked>
```
Comment (holistic-fox) [inline training-service/src/app/content/summary/summary.component.html]: value and checked are not needed attrs here

**Review by holistic-fox — CHANGES_REQUESTED** [review]

File: training-service/src/app/content/summary/summary.component.html (hunk 1 above)
Comment (D-Petrus) [reply training-service/src/app/content/summary/summary.component.html]: done

File: training-service/src/app/content/summary/summary.component.html (hunk 2 above)
Comment (D-Petrus) [reply training-service/src/app/content/summary/summary.component.html]: done

**Pull request merged**